### PR TITLE
FIX: Added fix if mongodb://localhost fails

### PR DESCRIPTION
**Pull request by drsimplegraffiti**
**Summary**

**Examples**

### CODE REVIEW

**Comment by Uzlopak** [comment]
That is imho quite radical approach when the issue is actually, that you should not use localhost but the IPv4 127.0.0.1 on node18

**Comment by Uzlopak** [comment]
I dont agree with this change. Also above your remark it says to use 127.0.0.1

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Mongoose is a [MongoDB](https://www.mongodb.org/) object modeling tool designed to work in an asynchronous environment. Mongoose supports both promises and callbacks.
 
-[![Slack Status](https://img.shields.io/badge/slack-mongoosejsteam-34D058.svg?logo=slack )](https://mongoosejsteam.slack.com)
+[![Slack Status](https://img.shields.io/badge/slack-mongoosejsteam-34D058.svg?logo=slack)](https://mongoosejsteam.slack.com)
```
Comment (vkarpov15) [inline README.md:5]: Why all these unrelated changes?

File: README.md [hunk 2]
```diff
@@ -91,22 +97,22 @@ const BlogPost = new Schema({
   author: ObjectId,
   title: String,
   body: String,
-  date: Date
+  date: Date,
```
Comment (vkarpov15) [inline README.md:100]: :-1:

File: README.md [hunk 3]
```diff
@@ -77,6 +77,12 @@ Once connected, the `open` event is fired on the `Connection` instance. If you'r
 
 **Note:** _If the local connection fails then try using 127.0.0.1 instead of localhost. Sometimes issues may arise when the local hostname has been changed._
 
+If `localhost` fails, use
```
Comment (vkarpov15) [inline README.md:80]: The remark above says to use `127.0.0.1`, which is better IMO.

File: README.md [hunk 4]
```diff
@@ -77,6 +77,12 @@ Once connected, the `open` event is fired on the `Connection` instance. If you'r
...
+
+```js
+await mongoose.connect('mongodb://0.0.0.0:27017/my_database');
```
Comment (vkarpov15) [inline README.md:83]: I'm pleasantly surprised that connecting to `0.0.0.0` actually works. However, I don't want to rely on that or recommend that because [0.0.0.0 is "used to designate an invalid, unknown or non-applicable target"](https://en.wikipedia.org/wiki/0.0.0.0).

**Review by vkarpov15 — COMMENTED** [review]
I appreciate the PR, but I don't think the suggestion is correct or helpful. Plus all the unnecessary and unrelated style changes.
